subseas_flex_fcst/config-gt.yaml

```diff
@@ -1,22 +1,24 @@
 # App set up
+server: 0.0.0.0
 
 # Forecast options
 results_path: /data/xchourio/data/ACToday/NextGen/Seasonal/GTM/PRCP_ENACTS/output
 y_transform: false
 start_regex: '[A-Z][a-z]{2}\d{4}'
 start_format_in: '%b%Y'
 start_format_out: '%b%Y'
-forecast_mu_file_pattern: NextGen_PRCPPRCP_CCAFCST_mu_*_mystartandlead.tsv
-forecast_var_file_pattern: NextGen_PRCPPRCP_CCAFCST_var_*_mystartandlead.tsv
-obs_file_pattern: CanSIPSv2_PRCPPRCP_CCAFCST_Obs_*_mystartandlead.txt 
-hcst_file_pattern: CanSIPSv2_PRCPPRCP_CCAFCST_xvPr_*_mystartandlead.txt
+forecast_mu_file_pattern: NextGen_PRCPPRCP_CCAFCST_mu_*_SLtarget.tsv
+forecast_var_file_pattern: NextGen_PRCPPRCP_CCAFCST_var_*_SLtarget.tsv
+obs_file_pattern: CanSIPSv2_PRCPPRCP_CCAFCST_Obs_*_SLtarget.txt 
+hcst_file_pattern: CanSIPSv2_PRCPPRCP_CCAFCST_xvPr_*_SLtarget.txt
 variable: Precipitation
 
 #S,L, target dates options
 target_period_length: 3
 time_units: months
 leads: #provider_ID:leadTime_value
     L1: 1
+targets: none
```

Comment:
Use `null` (the null value) instead of `"none"` (a string of four characters), here and in the other config files.